File: 2083-three-divisors/three-divisors.cpp
class Solution {
public:
    bool isThree(int n) {
        int c = 2;
        int s = n/2;
        for(int i=2; i<=s; ++i) {
            if(n%i==0) {
                ++c;
            }
        }
        return c==3;
    }
};

File: 2083-three-divisors/three-divisors.py
class Solution:
    def isThree(self, n: int) -> bool:
        counter = 2
        half = int(n/2)
        for i in range(2, half+1):
            if(n%i==0):
                counter = counter + 1
            
        if(counter==3):
            return True
        else:
            return False
        
        